projects/rero/ng-core/src/lib/record/editor/wrappers/toggle-wrapper/toggle-wrappers.component.ts

```diff
@@ -27,7 +27,7 @@ import { isEmpty, removeEmptyValues } from '../../utils';
           <input class="custom-control-input" type="checkbox" id="toggle-switch-{{ field.id }}" \
                  (change)="toggle($event)" [checked]="tsOptions.enabled">
           <label class="custom-control-label" for="toggle-switch-{{ field.id }}" \
-                 [tooltip]="tsOptions.description">{{ tsOptions.label }}</label>
+                 [tooltip]="tsOptions.description|translate" translate>{{ tsOptions.label }}</label>
```

Comment:
You need to remove the translate tag as it will put `{{ tsOptions.label }}` for translation. Use `{{ tsOptions.label | translate }}` instead

Reply:
This syntax work perfectly. This is new in TranslateModule

Reply:
Changed. Done